src/main/java/me/spartacus04/jext/listeners/JukeboxClickEvent.kt

```diff
@@ -30,9 +30,13 @@ internal class JukeboxClickEvent : JextListener() {
     }
 
     private fun defaultBehaviour(event: PlayerInteractEvent, block: Block) {
-        if(!INTEGRATIONS.hasJukeboxAccess(event.player, block)) {
-             event.isCancelled = true
-             return
+        if(INTEGRATIONS.hasJukeboxAccess(event.player, block)==false) {
+            event.isCancelled = true
+            return
+        }
```

Comment:
Just use the `!` before the check, this may not make sense now because you made the function nullable but we'll make it not nullable in the next few functions